src/systems/user_commands/UserCommands.cc

```diff
@@ -657,6 +759,115 @@ bool RemoveCommand::Execute()
   return true;
 }
 
+
+//////////////////////////////////////////////////
+LightCommand::LightCommand(msgs::Light *_msg,
+    std::shared_ptr<UserCommandsInterface> &_iface)
+    : UserCommandBase(_msg, _iface)
+{
+}
+
+//////////////////////////////////////////////////
+bool LightCommand::Execute()
+{
+  auto lightMsg = dynamic_cast<const msgs::Light *>(this->msg);
+  if (nullptr == lightMsg)
+  {
+    ignerr << "Internal error, null light message" << std::endl;
+    return false;
+  }
+
+  Entity lightEntity{kNullEntity};
+
+  if (lightMsg->id() != 0)
```

Comment:
Can you also compare to `kNullEntity`? Like here:

https://github.com/ignitionrobotics/ign-gazebo/blob/f6a7329ad7a283034a54647d44c4c061666cfeb8/src/systems/user_commands/UserCommands.cc#L681